src/System.Globalization/ref/System.Globalization.cs

```diff
@@ -15,88 +15,6 @@ public static partial class CharUnicodeInfo
         public static System.Globalization.UnicodeCategory GetUnicodeCategory(char ch) { return default(System.Globalization.UnicodeCategory); }
         public static System.Globalization.UnicodeCategory GetUnicodeCategory(string s, int index) { return default(System.Globalization.UnicodeCategory); }
     }
-    public partial class CompareInfo
```

Comment:
>public partial class CompareInfo [](start = 4, length = 32)

You will have to manually add type forwards to all of these types you are pushing down to System.Runtime. You can see an example for this in #10343